docs/pages/versions/unversioned/sdk/reanimated.mdx

```diff
@@ -20,36 +20,6 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
 
 <APIInstallSection href="https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation" />
 
-After the installation completes, you must also add the Babel plugin to **babel.config.js**:
-
-```js babel.config.js
-module.exports = function (api) {
-  api.cache(true);
-  return {
-    presets: ['babel-preset-expo'],
-    plugins: ['react-native-reanimated/plugin'],
-  };
-};
-```
-
-After you add the Babel plugin, restart your development server and clear the bundler cache: `npx expo start --clear`.
-
-> If you load other Babel plugins, the Reanimated plugin has to be the last item in the plugins array.
-
-### Web support
-
-**For web,** you'll have to install the [`@babel/plugin-proposal-export-namespace-from`](https://babeljs.io/docs/en/babel-plugin-proposal-export-namespace-from#installation) Babel plugin and update the **babel.config.js** to load it:
-
-{/* prettier-ignore */}
-```js babel.config.js
-plugins: [
-  '@babel/plugin-proposal-export-namespace-from',
-  'react-native-reanimated/plugin',
-],
-```
-
-After you add the Babel plugin, restart your development server and clear the bundler cache: `npx expo start --clear`.
```

Comment:
we should probably add something here to indicate that you can remove the reanimated babel plugin that we automatically add, and link to the babel-preset-expo readme or docs for that